hera_cal/apply_cal.py

```diff
@@ -131,6 +132,10 @@ def apply_cal(data_infilename, data_outfilename, new_calibration, old_calibratio
         add_to_history += ' OLD_CALFITS_HISTORY: ' + old_uvc.history + '\n'
         old_calibration, _ = io.load_cal(old_uvc)
 
+    # parse vis_units
```

Comment:
I don't think this needs to be an explicitly enumerated script argument of this function...just pass this in as part of the kwargs in the script. I think it would be better to make this part of the argparser and let the script take care of things.